rest/admin_api.go

```diff
@@ -317,10 +317,27 @@ func (h *handler) handleActiveTasks() error {
 func (h *handler) handleGetRawDoc() error {
 	h.assertAdminOnly()
 	docid := h.PathVar("docid")
+	includeDoc := h.getBoolQuery("include_doc")
+	redact := h.getOptBoolQuery("redact", true)
+	salt := h.getQuery("salt")
```

Comment:
Should the default salt be randomised on each call? I think that's how sgcollect_info does it for an undefined salt.